frontend/src/app/containers/CourseInput.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import  { useDispatch } from 'react-redux';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import uuid from 'uuid/v4'
+import { addCourseAction } from '../store/actions/addAction';
+
+export default () => {
+
+    const [title, setTitle] = useState('');
+    const [prof, setProf] = useState('');
+    const [domain, setDomain] = useState('');
+    const [nrOfLessons, setNrOfLessons] = useState('');
+    const [completed, setCompleted] = useState('');
+    const dispatch = useDispatch();
+    const addCourse = (course) => dispatch(addCourseAction(course))
+
+
+    const onChangeTitle = event => {
+        setTitle(event.target.value);
+    };
+
+    const onChangeProf = event => {
+        setProf(event.target.value);
+    };
+    const onChangeDomain = event => {
+        setDomain(event.target.value);
+    };
+    const onChangeNrOfLessons = event => {
+        setNrOfLessons(event.target.value);
+    };
+    const onChangeCompleted = event => {
+        setCompleted(event.target.value);
+    };
+
+    const onSubmitBtn = event => {
+        event.preventDefault();
+        addCourse({
+            id : uuid(),
+            title: title,
+            prof:  prof,
+            domain: domain,
+            nrOfLessons: nrOfLessons,
+            completed: completed
+        });
+    };
+
+    return (
+        <div className="course-input-container">
+            <Form onSubmit={onSubmitBtn}>
```

Comment:
you can put the form fields in another component

Reply:
I'll fix that soon